crates/sui/tests/protocol_version_tests.rs

```diff
@@ -305,7 +316,7 @@ mod sim_only_tests {
             }
         }));
 
-        monitor_version_change(&test_cluster, 2 /* expected proto version */).await;
+        monitor_version_change(&test_cluster, FINISH /* expected proto version */).await;
```

Comment:
Is it worth standardising on calls to `expect_upgrade_succeeded` and `expect_upgrade_failed`?